docs/faq.md

```diff
@@ -0,0 +1,13 @@
+# Frequently Asked Questions
+
+## Why are you charging for the game on Steam and iOS when it is available for free?
+
+The game is being charged for on Steam and iOS because it is one way of supporting Anuke. You have the choice of getting the game for free or not, just like in Itch.io. You can choose to get the Steam version or get the regular desktop release. Unfortunately this is not possible on the App Store [anuke you will know this]
+
+## How can I contribute to the game? 
+
+- The easiest way is to give constructive feedback and discussion (preferably about the game) in our Discord Server.
+- Report bugs that you find **on the Discord Server rather than the Issues section on the GitHub repository**. If you know how to program in Java, feel free to help Anuke out and fix said bugs, then open a PR. 
```

Comment:
> on the Discord Server rather than the Issues section on the GitHub repository

I don't know about this one. The Discord bug reporting system is painfully limited by virtue of being a single message with no direct way to reply. As for Github, it *should* be better, but it appears people have absolutely no idea what a template is or how to use it. I've closed at least 3 issues today from people who completely ignored the text it makes you fill out.

Reply:
Oof im sorry bout that one. Havent been on the discord in a while (probably never will be) so I don't know about the condition of bug reporting there. I will add instructions to use the template.